eslint.config.mjs

```diff
@@ -19,7 +20,14 @@ export default antfu({
     'style/quotes': 'off',
     'style/semi': 'off',
     'style/type-generic-spacing': 'off',
-    'ts/consistent-type-imports': 'off',
-    'ts/consistent-type-definitions': ['error', 'type'],
+  },
+  typescript: {
+    overrides: {
+      // https://github.com/antfu/eslint-config/issues/570
+      'ts/consistent-type-imports': [
+        'error',
+        { fixStyle: 'inline-type-imports' },
+      ],
+    },
```

Comment:
This override is separate at least until:

- https://github.com/antfu/eslint-config/issues/570